build.gradle.kts

```diff
@@ -14,7 +14,7 @@ allprojects {
     version = "1.5.0-SNAPSHOT"
 
     repositories {
-        mavenLocal()
+        //mavenLocal()
```

Comment:
Why is this commented?

Reply:
I added a comment in https://github.com/ACINQ/lightning-kmp/pull/457/commits/4798b41df3534a750c0474746de9da69bdfe4488 (we could also just remove this line)